### PR TITLE
Sequent calculi for negative modalities

### PR DESCRIPTION
**Pull request by yoni206**
A suggested entry to the encyclopedia that includes several sequent calculi for negative modalities.

### CODE REVIEW

File: Source/mainmatter/negmodal.tex [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+ {$\GPKF=\mostbasic+[\FUNC]+[\ineg=\uneg]$,}
+ {$\GPKB=\mostbasic+[\B_{1}]+[\B_{2}]$,}
+ {$\GPKFour=\mostbasic+[\KFour_{1}]+[\KFour_{2}]$,}
+ {$\GPKDB=\GPKB+[\Di_{\B}]$,} and
+ {$\GPKFourD=\GPKFour+[\Di_{\KFour}]$.}
+
+ \end{clarifications}
+
+ \begin{history}
+$\GPK$ was introduced in \cite{dod:mar:ENTCS2013}. All other systems were introduced in \cite{lahav_it_2016}.
```
Comment (ceilican) [inline Source/mainmatter/negmodal.tex]: Thanks for the contribution, Yoni!

Could you add one or two short sentences here or in the clarification section, explaining and motivating negative modalities? (I know that sequent calculus experts will be able to figure this just by looking at the inference rules, but the encyclopedia's audience is a bit wider.)

File: Source/mainmatter/negmodal.tex [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+ \end{history}
+
+ \begin{technicalities}
+All systems enjoy cut-admissibility, besides
```
Comment (ceilican) [inline Source/mainmatter/negmodal.tex]: If space permits, could you briefly say how cut-admissibility was proven?

File: Source/mainmatter/negmodal.tex [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+those that contain the rule $\B$, that are analytic.
```
Comment (ceilican) [inline Source/mainmatter/negmodal.tex]: This sentence ("All systems enjoy cut-admissibility, besides those that contain the rule $\B$, that are analytic.") is a little bit ambiguous/confusing/unclear. Do you mean something like this: "All systems are analytic. All systems besides those containing the rule $\B$ enjoy cut-admissibility"?   Could you try to reformulate and improve it?

**Comment by ceilican** [comment]
Thanks again!

**Comment by yoni206** [comment]
Thank you for all the comments.
Any other suggestions are welcome.

**Comment by ceilican** [comment]
Thanks, Yoni!
I accepted the pull request.

I fixed one thing: "Kleene" --> "Kleene's".  Do you approve my correction?

Also, the date of the calculus is stated as 2016, but the dates of the publications are different. If this is intentional (as I expect), it is ok. But I just wanted to check that it is not a mistake. Is it intentional?

**Comment by yoni206** [comment]
Hi Bruno,
Thanks for the fixes!

Yeah, sure "Kleene's" is good.
The thing with the date is as follows:
PK is from 2014. All the rest are from a paper in AiML 2016. However, I thought it would be better to cite the extended version of that paper, that was just published.
If you think this can be handled in a better way, please let me know.

**Comment by ceilican** [comment]
Considering that you still have enough blank space in the page, I suggest that you cite both the AiML paper and the extended paper. It is up to you.

**Comment by yoni206** [comment]
Hi Bruno, I've made a new pull request with your suggestion.

Thanks!

Yoni.

**Pull request merged**